### ISSUE TITLE
Too many phones... and this is not just for translators!

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#2471]
#### Issue

There are too many "phone numbers" labels in language files. Phone numbers are a key field for CRM users but there are too many variants of the labels (30 of them) in different modules.

_On recording details from a contact_
I really doubt there can be a consistency when registering phone numbers, even for english users.
Why is there a Any / Alternate / Other phone? Values will be stored in the same table field or not? 
Or where should I place a business phone: Work or Office? 

_On the translations_
Its almost impossible to maintain translations consistency keeping it related with the contents used in SuiteCRM database fields.
Not only there are similar names but also there are many variants in each label. This causes us to translate each one when it would be easy to just translate 3 of them (Crowdin auto-fill same value translation). Only we have to translate 30 variants.

Note: As told before this is not just a language issue but a functionality issue as nobody really now where to fill a given number when they change modules.
#### Expected Behavior

All phone labels should be clear and with same usage on all modules. 
#### Actual Behavior

Too many variations for labels across all modules. 
30 different labels in total! See the list:

```
Phone:
Phone Number
Primary Phone
Contact Phone:

Office Phone:
Office Phone
Phone Office:
Work Phone:
Work Phone
Phone Work

Home Phone:
Phone Home

Any Phone:
Any Phone
Alternate Phone:
Phone Alternate

Other Phone:
Other Phone
Phone Other
Other:

Assistant Phone:
Assistant Phone

Mobile Phone
Phone Mobile
Mobile:
Mobile

Phone Fax:
Phone Fax
Fax:
Fax
```
#### Possible Fix

To reduce the labels variations it would require a wiki/data sheet for fields specification and labels usage so modules developers can use it as reference. This is a big but important task.
So why not start this work with Phone labels? "Email Address" value would be next even if it has less variations on it.
@salesagility please start simple, with just a little page, and then go from there to other labels!
#### Your Environment

- SuiteCRM Version used: 7.7.6

**Comment by gunnicom** [comment]
Alternative a phone number could be a seperate object like an email address.
You can then add as many numbers as you need to one person, with each having a dropdown for the type "office", "fax", "mobile" ...

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Too many phones... and this is not just for translators!](https://trello.com/c/jClMrqsX/60-too-many-phones-and-this-is-not-just-for-translators)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
